core/src/test/java/org/frankframework/http/authentication/OAuthAccessTokenKeycloakTest.java

```diff
@@ -36,7 +36,7 @@
 public class OAuthAccessTokenKeycloakTest extends SenderTestBase<HttpSender> {
 
 	@Container
-	static final KeycloakContainer keycloak = new KeycloakContainer().withRealmImportFile("/Http/Authentication/iaf-test.json");
+	static final KeycloakContainer keycloak = new KeycloakContainer("quay.io/keycloak/keycloak:26.0.5").withRealmImportFile("/Http/Authentication/iaf-test.json");
```

Comment:
Gaat er iets mis in een minor update van keycloak? Ik zou eerder verwachten dat er iets met een major misgaat.

Reply:
Het rare is dat hij zegt al mis te gaan bij het opstarten van de container. Dus nog niet eens bij het uitvoeren van tests.